src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -107,7 +107,7 @@ export function isPythonNotebook(metadata?: nbformat.INotebookMetadata) {
     }
 
     // Valid notebooks will have a language information in the metadata.
-    return kernelSpec?.language === PYTHON_LANGUAGE;
+    return kernelSpec?.language === PYTHON_LANGUAGE || metadata?.language_info?.name === PYTHON_LANGUAGE;
```

Comment:
This was the fix.

Reply:
Does the isPythonNotebook helper function work here?

Reply:
This is the `isPythonNotebook` helper function (hence its a yes)

Reply:
Teaches me to not be careful where I was looking. I thought this was an independent check that wasn't using the helper.

Reply:
But I'm glad you are ensuring (in code reviews) we have a common functions instead of duplicating the logic elsewhere..
So, kudos to you, awesome reviewing...